docs/destinations/forwarding-events/event-forwarding-monitoring-and-troubleshooting/index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "Monitoring and troubleshooting forwarders"
+description: "Monitor event forwarder performance, debug failures, and understand retry logic with cloud metrics, failed event logs, and BDP Console statistics."
+sidebar_position: 15
+---
+
+This page is an overview of how to monitor event forwarder performance and diagnose delivery issues. Snowplow provides both summary metrics and detailed failed event logs, to help you understand failure patterns and troubleshoot specific problems.
+
+## Failure types and retry logic
+
+Forwarders uses the same retry logic and failure handling as the underlying [Snowbridge failure model](/docs/api-reference/snowbridge/concepts/failure-model/index.md). Snowplow handles event forwarding failures differently depending on the type:
+
+- **Invalid data failures**: Snowplow treats events that fail transformation or violate destination API requirements as unrecoverable. Event forwarding creates [event forwarding error failed events](https://iglucentral.com/?q=event_forwarding_error) and logs them in your cloud storage bucket without retry.
+- **Transformation failures**: Snowplow treats JavaScript transformation errors as invalid data and logs them to your cloud storage bucket without retry.
```

Comment:
```suggestion
- **Transformation failures**: Where the mapping transformation encounters an error, event forwarding creates [event forwarding error failed events](https://iglucentral.com/?q=event_forwarding_error) and logs them in your cloud storage bucket without retry. 
```

Possibly add something like: "There are safeguards against deploying with such errors, but for example custom transformation may sometimes hit errors at runtime."